packages/core/lib/services/logger.js

```diff
@@ -1,11 +1,9 @@
-
-
 export default {
-  name: "logger",
+  name: 'logger',
```

Comment:
This is the json rpc client for the logger, it will be replaced by a custom winston transport in an additional PR.
Added here already https://www.notion.so/dockteam/Wallet-sdk-code-review-25d2f2d7644c470ea5a1012a8cd7c871